internals/secrethub/read_test.go

```diff
@@ -1,9 +1,130 @@
 package secrethub
 
 import (
+	"os"
 	"testing"
+	"time"
+
+	"github.com/secrethub/secrethub-cli/internals/cli/clip/fakeclip"
+	"github.com/secrethub/secrethub-cli/internals/cli/filemode"
+	"github.com/secrethub/secrethub-cli/internals/cli/ui/fakeui"
+	"github.com/secrethub/secrethub-go/internals/api"
+	"github.com/secrethub/secrethub-go/internals/assert"
+	"github.com/secrethub/secrethub-go/internals/errio"
+	"github.com/secrethub/secrethub-go/pkg/secrethub"
+	"github.com/secrethub/secrethub-go/pkg/secrethub/fakeclient"
 )
 
 func TestReadCommand_Run(t *testing.T) {
-	// TODO SHDEV-1029 Test ReadCommand.
+	testErr := errio.Namespace("test").Code("test").Error("test error")
+	testSecret := []byte("Remember! Reality's an illusion, the universe is a hologram, buy gold! Bye! - Bill Cipher")
```

Comment:
Though I'm not into Gravity Falls (am I missing out on something?), gotta love the reference.

![](https://media0.giphy.com/media/l3q2LH45XElELRzRm/giphy.gif)

Reply:
You got it right ^-^